aspnetcore-features/MapControllersVsMapControllerRouteSln/Tests/ConventionIntegrationTest.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Mvc.Testing;
+
+namespace Tests;
+
+public class ConventionIntegrationTest: IClassFixture<WebApplicationFactory<Program>>
+{
+    private readonly WebApplicationFactory<Program> _factory;
+
+    public ConventionIntegrationTest(WebApplicationFactory<Program> factory)
+    {
+        _factory = factory;
+    }
+    
+    [Theory]
+    [InlineData("/")]
+    [InlineData("/Home")]
+    [InlineData("/Home/Index")]
+    public async Task WhenMapControllerRouteRun_ThenSuccess(string url)
```

Comment:
We should test all controllers I would say.

Reply:
The "Tests" project covers route testing for both controllers.